javascript/javascript3/week2/Rewrite-time.js

```diff
@@ -0,0 +1,40 @@
+
+/*function setTimeoutPromise(givenTime){
+  console.log('I was called immediately');
+  const promise = new Promise((resolve,reject)=>{
+    //const settingTime = givenTime;
+    setTimeout(()=>{
+      resolve(givenTime);
+    },givenTime)
+  });
+  return promise;
+}
+
+
+setTimeoutPromise(9000)
+.then((givenTime) => {
+  console.log(`I was loggedOut after ${givenTime}`);
+})
+.catch((err)=>{
+  console.log('error');
+});*/
+
+
+function getCurrentLocation(){
+  const geolocationPromise = new Promise((resolve,reject)=>{
+  navigator.geolocation.getCurrentPosition((position)=>resolve(position), (error)=>reject(error));
+});
+return geolocationPromise();
```

Comment:
`geolocationPromise` is not a function to call it!

Reply:
ohh sorry yes I will rectify